tests/wp-unit/tests/include/Core/Posts/Metabox/DeletePostsByCategoryMetaboxTest.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Posts\Metabox;
+
+use BulkWP\Tests\WPCore\WPCoreUnitTestCase;
+
+/**
+ * Test Deletion of posts by category.
+ *
+ * Tests \BulkWP\BulkDelete\Core\Posts\Metabox\DeletePostsByCategoryMetabox
+ *
+ * @since 6.0.0
+ */
+class DeletePostsByCategoryMetaboxTest extends WPCoreUnitTestCase {
+
+	/**
+	 * The metabox that is getting tested.
+	 *
+	 * @var \BulkWP\BulkDelete\Core\Posts\Metabox\DeletePostsByCategoryMetabox
+	 */
+	protected $metabox;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->metabox = new DeletePostsByCategoryMetabox();
+	}
+
+	public function test_for_deleting_posts_from_all_categories_default_post_type(){
```

Comment:
I am still seeing code formatting issues. Look at the `{` in this function and other functions in this file.

Reply:
Fixed